index.d.ts

```diff
@@ -1,8 +1,18 @@
+interface Options {
+    /**
+    Provide filter function for modules included in clearing
+
+    @default undefined
+	*/
+	readonly filter?: (fullPath: string) => boolean | undefined
```

Comment:
```suggestion
	readonly filter?: (fullPath: string) => boolean
```

Reply:
If i take out "undefined" i would have to rework the javascript because thats not how the function works

Reply:
replaced undefined with void, maybe that is better for you?

Reply:
@dl748 I suspect Sindre will prefer you rework the javascript then. A filter function really should either return `true` or `false`. If you wrote code assuming that the user could return `undefined` from it, then change your code to stop assuming that.

Reply:
What @papb said.

Reply:
Thats fine if thats whats wanted.  I just felt that in JS, if a dev put in a noop function, that it would work normally as well. `clearModule('./my-module', { filter: ()=>{} })`